src/affine.rs

```diff
@@ -156,81 +156,99 @@ impl Affine {
         aff.pre_translate(-point.to_vec2())
     }
 
-    /// A rotation by `th` followed by `self`.
+    /// A [rotation] by `th` followed by `self`.
     ///
     /// Equivalent to `self * Affine::rotate(th)`
+    ///
+    /// [rotation]: Affine::rotate
     #[inline]
     #[must_use]
     pub fn pre_rotate(self, th: f64) -> Self {
         self * Affine::rotate(th)
     }
 
-    /// A rotation by `th` about `center` followed by `self`.
+    /// A [rotation] by `th` about `center` followed by `self`.
     ///
     /// Equivalent to `self * Affine::rotate_about(th)`
+    ///
+    /// [rotation]: Affine::rotate_about
     #[inline]
     #[must_use]
     pub fn pre_rotate_about(self, th: f64, center: Point) -> Self {
         Affine::rotate_about(th, center) * self
```

Comment:
I think this needs to be:

```suggestion
        self * Affine::rotate_about(th, center)
```
Although that might be for a different PR

Reply:
This needs to be a separate PR. I think the core is wrong here, not the doc comment.

Reply:
What I meant was that I too think the code is wrong and this is a docs patch, so I'd rather handle it separately. :)